great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -16,6 +16,9 @@
 from great_expectations.core.config_peer import ConfigPeer
 from great_expectations.core.expectation_configuration import ExpectationConfiguration
 from great_expectations.core.expectation_suite import ExpectationSuite
+from great_expectations.core.usage_statistics.usage_statistics import (
+    usage_statistics_enabled_method,
+)
```

Comment:
This currently does nothing but is here to prove that we can now import usage stats into RBP without an issue (at least with the pattern noted above).

This will be used in GREAT-545 but is currently not touched in the file. Let me know if you'd like to me to remove this.